src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +77,31 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  public boolean getIsGroup() {
+    return isGroup;
+  }
+
+  public void addParticipant(UUID id) {
+    if(isGroup == true){
```

Comment:
`if (isGroup) {`
Fix spacing and you don't need to explicitly say true/false for booleans like that